h2o-r/h2o-package/R/kvstore.R

```diff
@@ -76,6 +76,10 @@ h2o.removeAll <- function(timeout_secs=0, retained_elements = c()) {
         retained_keys <- append(retained_keys, element@model_id)
       } else if (is.H2OFrame(element)) {
         retained_keys <- append(retained_keys, h2o.getId(element))
+      } else if( is.character(element) ) {
+        retained_keys <- append(retained_keys, element)
+      } else {
+        stop("The 'retained_elements' variable must be a Keyed instance (e.g. H2OFrame, H2OModel) or character")
```

Comment:
I think the naming in this method is probably not consistent with rest of the package. On the client-side we usually don't use term "keys", we use "ids" instead. See model id, frame id, functions like h2o.getFrame, h2o.getId...

Thankfully the API of this method calls for "retained_elements" and doesn't use the "key" terminology.

I think we should
- fix the documentation of this method to use "ids" instead of "keys" where appropriate. 
- since we are allowing also ids/keys as a parameter the documentation should reflect that.
- error message should be imho simplified to something like "... be either an  instance of H2OFrame/H2OModel or an id of H2OFrame/H2OModel.

cc: @Pscheidl

Reply:
That is corrected, I used the same terminology as the Python client uses to unify it - definitely should've been adapted to R conventions we've got there so far.

Reply:
Documentation is updated. I slightly update the documentation for h2o.rm also